storage/src/vespa/storage/storageserver/communicationmanager.cpp

```diff
@@ -273,8 +273,9 @@ struct PlaceHolderBucketResolver : public BucketResolver {
     virtual document::BucketSpace bucketSpaceFromName(const vespalib::string &) const override {
         return document::BucketSpace::placeHolder();
     }
-    virtual vespalib::string nameFromBucketSpace(const document::BucketSpace &) const override {
-        return "";
+    virtual vespalib::string nameFromBucketSpace(const document::BucketSpace &bucketSpace) const override {
+        assert(bucketSpace == spi::FixedBucketSpaces::default_space());
+        return "default";
```

Comment:
Could also return `spi::FixedBucketSpaces::to_string(bucketSpace)` to avoid having to hard-code the default bucket space string